app/services/container_topology_service.rb

```diff
@@ -80,10 +80,10 @@ def entity_display_type(entity)
           "Container" # the container entity itself
         end
       else
-        if name.eql? "Vm"
-          'VM'
+        if entity.kind_of?(Vm)
+          name.upcase # turn Vm to VM because it's an abbreviation
```

Comment:
yes, it is for display purposes.
VM is the only entity that is an abbreviation in the topology, so for all the rest this is irrelevant at the moment. btw what other entities are an abbreviation in miq? can't think of any. (for generic purposes)